onnxscript/ir/passes/_pass_infra.py

```diff
@@ -180,23 +180,31 @@ class InPlacePass(PassBase):
     """A pass that modifies the input model in place and returns it."""
 
     @property
-    def in_place(self) -> bool:
+    @final
+    def in_place(self) -> Literal[True]:
```

Comment:
Maybe just turn these to be class attributes, so does in PassBase? We don't need them in runtime anyway.